shell/platform/windows/flutter_windows_view.cc

```diff
@@ -84,10 +84,13 @@ void UpdateVsync(const FlutterWindowsEngine& engine,
 }  // namespace
 
 FlutterWindowsView::FlutterWindowsView(
+    int64_t view_id,
```

Comment:
Not too sure how many usages of this already exist in the windows embedder, but at least for internal API, consider a `using FlutterViewId = int64_t;` or similar for readability and greppability of the code, i.e. for the same reason we define `kImplicitViewId`.

Reply:
Added!